### PR TITLE
feat(locales): latin translation

### PR DESCRIPTION
**Pull request by fernandoamz**
## What does this PR do?
Add translation and files to load Spanish Lat.

## Test Plan
Someone who can run the project should make a checkout to review if the files and translations for Spanish lat are running correctly.

### Have you read the [Contributing Guidelines on issues](https://github.com/appwrite/appwrite/blob/master/CONTRIBUTING.md)?

(Write your answer here.)

### CODE REVIEW

File: app/config/locale/translations/la.json [hunk 1]
```diff
@@ -0,0 +1,232 @@
...
+  "emails.magicSession.signature": "{{project}} equipo",
+  "emails.recovery.subject": "Recuperar contraseña",
+  "emails.recovery.hello": "Hola {{name}}",
+  "emails.recovery.body": "Sigue este vínculo para recuperar la contraseña de tu proyecto {{project}}",
+  "emails.recovery.footer": "Si no solicitaste recuperar tu contraseña, Puedes ignorar este mensaje",
+  "emails.recovery.thanks": "Gracias",
+  "emails.recovery.signature": "{{project}} equipo",
+  "emails.invitation.subject": "Invitacion para %s en el equipo %s",
+  "emails.invitation.hello": "Hola",
+  "emails.invitation.body": "Este correo electrónico te fue enviado por que el dueño del proyecto quiere invitarte a convertirte en miembro en el equipo {{team}} en el {{project}}",
```
Comment (christyjacob4) [inline app/config/locale/translations/la.json:26]: https://github.com/appwrite/appwrite/blob/master/app/config/locale/translations/en.json#L26

You're missing the `{{owner}}` placeholder

**Review by christyjacob4 — CHANGES_REQUESTED** [review]
Please address the comments

**Comment by christyjacob4** [comment]
It looks like you have translated this to Spanish. Whereas the issue was to translate it to Latin `la` .

**Comment by fernandoamz** [comment]
Hey @christyjacob4, Sorry for confusing I thought it was Spanish Latino, I am doing the translation to Latin, I will push the new changes in a few moments.

**Comment by christyjacob4** [comment]
Sure! All the best :+1:

**Pull request closed without merging**